mozdownload/scraper.py

```diff
@@ -639,7 +639,9 @@ def get_build_info(self):
             self.build_index = 0
             self.logger.info('Selected build: build%s' % self.build_number)
         else:
-            self.logger.info('Selected build: build%s' % self.build_index)
+            # Otherwise displays incorrect build number
+            display_index = str(int(self.build_index) + 1)
+            self.logger.info('Selected build: build%s' % display_index)
```

Comment:
*sigh* Another one of those school boy errors of mine. @whimboo as an explanation. The following happened. I wrote the line

>self.logger.info('Selected build: build%d' % self.build_index + 1)

Which threw the following error:

>  INFO | Retrieving list of candidate builds from https://ftp.mozilla.org/pub/mozilla.org/firefox/candidates/36.0-candidates/
  INFO | Found 2 builds: build1, build2
Traceback (most recent call last):
  File "./scraper.py", line 1243, in <module>
    cli()
  File "./scraper.py", line 1235, in cli
    build = BUILD_TYPES[options.type](**kwargs)
  File "./scraper.py", line 628, in __init__
    Scraper.__init__(self, *args, **kwargs)
  File "./scraper.py", line 144, in __init__
    self.get_build_info()
  File "./scraper.py", line 643, in get_build_info
    self.logger.info('Selected build: build%d' % self.build_index + 1)
TypeError: cannot concatenate 'str' and 'int' objects

Consequently, I assumed that `self.build_index` is a string.

What do we learn from this? kids, don't code when you are tired ;)